demo/config.yaml

```diff
@@ -20,3 +20,11 @@ lcr-modules:
         inputs:
             sample_bam: "data/{sample_id}.bam"
             sample_bai: "data/{sample_id}.bam.bai"
+
+    strelka:
+        inputs:
+            sample_bam: "data/{sample_id}.bam"
+            sample_bai: "data/{sample_id}.bam.bai"
+            candidate_small_indels_output: "results/manta-2.1/99-outputs/vcf/{seq_type}--{genome_build}/candidateSmallIndels/{tumour_id}--{normal_id}--{pair_status}.candidateSmallIndels.vcf"
```

Comment:
Can this line be removed now?

Reply:
No because I need to call the final output for candidatesmallindels, instead of the one generated by `_manta_run`. I think it's because the output is a directory so we'll get Missing InputException if `input.indels` is set to the vcf of tbi files that are in `02-manta`